daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SExpr.scala

```diff
@@ -135,9 +135,33 @@ object SExpr {
     }
   }
 
+  // A fully saturated builtin application
+  final case class SEAppB(builtin: SBuiltin, args: Array[SExpr])
```

Comment:
Why not call is `SEAppSaturatedBuiltin`?

Reply:
I just saw your comment on the last PR, and made a change to improve the names/doc here

Reply:
your suggested name is even better. I used it